### PR TITLE
encode/decode access from FMElfinderBundle Loader (2/2) 

### PR DESCRIPTION
**Pull request by Ronan-Lenor**
The clean version.

Issue: https://github.com/helios-ag/FMElfinderBundle/issues/169

part 1/2: https://github.com/helios-ag/ElFinderPHP/pull/32

### CODE REVIEW

**Comment by Ronan-Lenor** [comment]
i really don't understand that error : 
" StyleCI — The StyleCI analysis has failed - 1 file needs addressing " ???????

**Comment by helios-ag** [comment]
Hah, actually you can ignore styleCI checks, because it allow to make automatic PR with fixes :)

File: Loader/ElFinderLoader.php [hunk 1]
```diff
@@ -25,6 +25,11 @@ class ElFinderLoader
     protected $configurator;
 
     /**
+     * @var ElFinderBridge
+     */
+    protected $Bridge;
```
Comment (helios-ag) [inline Loader/ElFinderLoader.php]: please use camelCase for naming variables

**Comment by Ronan-Lenor** [comment]
Is it good?

File: Loader/ElFinderLoader.php [hunk 2]
```diff
@@ -81,4 +90,41 @@ public function setConfigurator($configurator)
...
+     **/
+    public function encode(Request $request, $path)
+    {
+        $target = ($request->isMethod('POST')) ? $request->get('target') : $request->query->get('target');
+
+        if (empty($target)) {
+            throw new Exception('Request: target parameter is empty. Volume id can\'t be found.');
+        }
+
+        $Volume = $this->bridge->getVolume($target);
```
Comment (helios-ag) [inline Loader/ElFinderLoader.php]: same here :)

File: Loader/ElFinderLoader.php [hunk 3]
```diff
@@ -81,4 +90,41 @@ public function setConfigurator($configurator)
...
+    /**
+     * Decode path from hash.
+     *
+     * @param string $hash
+     * 
+     * @return string
+     **/
+    public function decode($hash)
+    {
+        $Volume = $this->bridge->getVolume($hash);
```
Comment (helios-ag) [inline Loader/ElFinderLoader.php]: and this one

**Comment by Ronan-Lenor** [comment]
Well,' i'm used to put a uppercase on an Object :P 
sould be ok now, i guess lol

**Comment by helios-ag** [comment]
Well what can i say... finally :)

**Pull request merged**